hibike/travis/Makefile

```diff
@@ -0,0 +1,14 @@
+### Build targets for Travis.
+.PHONY: install artifacts-install lint test
+
+install:
+	cd .. && sudo bash setup.sh
+
+artifacts-install:
+	$(nop)
+
+lint:
+	$(nop)
+
+test:
+	$(nop)
```

Comment:
Does it work if you add `cd .. && rm -rf ./bin && make DEVICE=ExampleDevice` here?